packages/editor/src/managers/clipboard/paste-manager.ts

```diff
@@ -255,6 +255,9 @@ export class PasteManager {
             this._addBlocks(blocks.slice(0, 1), parent, index, addBlockIds);
             this._editor.page.deleteBlock(lastBlockModel);
           } else {
+            if (currentSelectionInfo.type === 'Range') {
+              selectedBlock?.text?.delete(0, selectedBlock.text.length);
```

Comment:
Delete the block if and only if you select the whole text

Reply:
Would I be able to achieve the above effects if I used this method? I'm trying to use `deleteModelsByRange` but it still seems to have some problems

![image](https://user-images.githubusercontent.com/50788736/212537111-228c99e7-3f40-49f0-b8f2-2e765a427a7b.png)

Reply:
I think you just need to delete the section part and insert